V2.16-RUN-15152-alertmanager-alerts (#633)
run-ai/docs: docs/admin/troubleshooting/alertmanager/RunaiContainerRestarting.md

---
title: RunaiContainerRestarting 
summary: This article describes the RunaiContainerRestarting alert.
authors:
    - Jason Novich
    - Viktor Koukouliev
date: 2024-Jan-10
---

## Meaning

> The `runai` container has restarted more than twice in the last 10 minutes.

Reviewer: ```suggestion
A Run:ai container has restarted more than twice in the last 10 minutes.
```

## Impact

> The container may be unavailable and affect Run:ai system.

Reviewer: ```suggestion
The container may be unavailable and affect Run:ai system feature functionalities.
```

## Severity

Warning

## Diagnosis

> To diagnose the issue. run `kubectl get pods -n runai kubectl get pods -n runai-backend`.

Reviewer: ```suggestion
To diagnose the issue, and the pods with issues, run `kubectl get pods -n runai kubectl get pods -n runai-backend`.
```

> The expected result should be one or more of the pods will have restart count >= 2.

Reviewer: ```suggestion
The expected result should be one or more pods where the restart count >= 2.
```
